sections/clause_06_overview.adoc

```diff
@@ -50,14 +50,14 @@ For the Things whose location changed, the HistoricalLocations entities offer th
 A Thing also can have multiple Datastreams.
 A Datastream is a collection of Observations grouped by the same ObservedProperty, Sensor and optionally by Feature and ObservingProcedure.
 An Observation is an event performed by a Sensor that produces a result whose value is an estimate of an ObservedProperty of any given Feature which may be a proximate or ultimate FeatureofInterest.
-Details of each above described entity are provided in <<sensing-entities1>>.
+Details of each above described entity are provided in <<sensing-core>>.
 
 The Sampling part is a new addition to the SensorThings API.
 As is often the case, an Observation may not be a true representation of the intended Feature's Property that an Observer may be trying to Observe.
 Sampling is a common strategy to understand the characteristics of an otherwise difficult-to-measure Property of any feature-of-interest.
-In order to generate Samplings, a Sampler, that may be any physical device or even a human being part of a survey campaign, must carefully select a SamplingProcedure.
-Samplings may be generated by a sequence of SamplingProcedures (and vice-versa), however a Sampler must employ a unique SamplingProcedure to maintain unique Sampling-Sampler-SamplingProcedure relationships.
-In scenarios where a Feature is not directly available for Sampling, a PreparationProcedure composed of multiple PreparationSteps may optionally be used to generate a PreparedFeature.
+In order to generate Samplings, a Sampler, that may be any physical device (or even a human being part of a survey campaign), must carefully select a SamplingProcedure.
+A Sampling may be generated by a SamplingProcedure. This SamplingProcedure can be used by multiple Samplers, however a Sampler can not use multiple SamplingProcedures. Hence, any Sampling that is generated by a Sampler is always associated with a unique SamplingProcedure.
```

Comment:
One sentence per line please, hard to comment otherwise.
> however a Sampler can not use multiple SamplingProcedures

Hmm, that may need fixing. I think some Samplers can implement many SamplingProcedures... However, a Sampling can only be created by a single one.

Reply:
Yeah, I agree with the Sampler->SamplingProcedure relationship. I was planning to discuss it in the telco today, good that you pointed this one out